### PR TITLE
Add a placeholder doc

### PR DESCRIPTION
**Pull request by gnufied**
Move online volume expansion to beta.

xref : https://github.com/kubernetes/enhancements/issues/531

### CODE REVIEW

File: content/en/docs/concepts/storage/persistent-volumes.md [hunk 1]
```diff
@@ -226,15 +226,9 @@ CSI volume expansion requires enabling `ExpandCSIVolumes` feature gate and also
...
-the `PersistentVolumeClaim` in ReadWrite mode. Therefore, if a pod or deployment is using a volume and
-you want to expand it, you need to delete or recreate the pod after the volume has been expanded by the cloud provider in the controller-manager. You can check the status of resize operation by running the `kubectl describe pvc` command:
-
-```
-kubectl describe pvc <pvc_name>
-```
-
-If the `PersistentVolumeClaim` has the status `FileSystemResizePending`, it is safe to recreate the pod using the PersistentVolumeClaim.
+When a volume contains a file system, the file system is only resized when a new Pod is using
+the `PersistentVolumeClaim` in ReadWrite mode. File System expansion is either done when Pod is starting up
```
Comment (sftim) [inline content/en/docs/concepts/storage/persistent-volumes.md]: Here's two suggestions; I hope they're useful:
```suggestion
the `PersistentVolumeClaim` in ReadWrite mode. File system expansion is either done when Pod is starting up
```

<br />

or possibly:
```suggestion
the PersistentVolumeClaim in `ReadWrite` mode. File system expansion is either done when a Pod is starting up
```
(closer to style guide)

File: content/en/docs/concepts/storage/persistent-volumes.md [hunk 2]
```diff
@@ -243,14 +237,15 @@ The FlexVolume can be resized on pod restart.
 
 #### Resizing an in-use PersistentVolumeClaim
 
-Expanding in-use PVCs is an alpha feature. To use it, enable the `ExpandInUsePersistentVolumes` feature gate.
+Expanding in-use PVCs is a beta feature and is enabled by default via `ExpandInUsePersistentVolumes` feature gate.
```
Comment (msau42) [inline content/en/docs/concepts/storage/persistent-volumes.md]: Do we need `{{< feature-state for_k8s_version="v1.15" state="beta" >}}`?

Comment (msau42) [reply content/en/docs/concepts/storage/persistent-volumes.md]: Also do we need to update the feature-gates.md page?

**Pull request merged**